### PR TITLE
docs: configure DHIS2 Maps with GeoJSON coordinates

### PR DESCRIPTION
**Pull request by turban**
This PR will update the user documentation about the new GeoJSON importer: 
1. Import/Export app
2. Configure DHIS2 Maps

Many of the changes are linting fixes.

### CODE REVIEW

File: src/user/configure-the-gis-app.md [hunk 1]
```diff
@@ -1,94 +1,198 @@
...
+[site](http://www.epsg-registry.org/). If you have already reprojected
+the geographic data to the geographic latitude/longitude (EPSG:4326) system,
+there is no need to explicitly define the output coordinate system, assuming
+that `ogr2ogr` can determine the input spatial reference system. Note that
+most GeoJSON files are using the EPSG:4326 system. You can determine the
+spatial reference system by executing the following command.
+
+    ogrinfo -al -so filename.shp
+
+These command assumes your geospatial data is in ESRI Shapefile (.shp) format,
```
Comment (jenniferarnesen) [inline src/user/configure-the-gis-app.md]: These -> This

File: src/user/configure-the-gis-app.md [hunk 2]
```diff
@@ -1,94 +1,198 @@
...
+<http://fwtools.maptools.org/>and download "FWTools", install it and open
+up the FWTools command shell. During the format conversion we also want
+to ensure that the output has the correct coordinate projection
+(called EPSG:4326 with geographic longitude and latitude). For a more
+detailed reference of geographic coordinates, please refer to this
+[site](http://www.epsg-registry.org/). If you have already reprojected
+the geographic data to the geographic latitude/longitude (EPSG:4326) system,
+there is no need to explicitly define the output coordinate system, assuming
+that `ogr2ogr` can determine the input spatial reference system. Note that
+most GeoJSON files are using the EPSG:4326 system. You can determine the
```
Comment (jenniferarnesen) [inline src/user/configure-the-gis-app.md]: are using -> use

Comment (Philip-Larsen-Donnelly) [reply src/user/configure-the-gis-app.md]:
```suggestion
most GeoJSON files use the EPSG:4326 system. You can determine the
```

File: src/user/configure-the-gis-app.md (hunk 1 above)
Comment (Philip-Larsen-Donnelly) [inline src/user/configure-the-gis-app.md]:
```suggestion
This command assumes your geospatial data is in ESRI Shapefile (.shp) format,
```

**Review by Philip-Larsen-Donnelly — APPROVED** [review]

**Pull request merged**